astropy/coordinates/builtin_frames/cirs_observed_transforms.py

```diff
@@ -40,7 +40,8 @@ def cirs_to_altaz(cirs_coo, altaz_frame):
     xp, yp = get_polar_motion(obstime)
 
     #first set up the astrometry context for CIRS<->AltAz
-    astrom = erfa.apio13(obstime.jd1, obstime.jd2,
+    obstime_utc = obstime if obstime.scale == 'utc' else obstime.utc
```

Comment:
@taldcroft - since `Time` instances are immutable, I think we could at least do the very simple change of just returning `self` if the scale is already correct.  The caching I'm less sure about; maybe adding a bit too much complexity.